README.md

```diff
@@ -5,200 +5,204 @@ test cases
+To configure a standard job and add a post-build action: 
+
+1\. Click **New Item** in the main Jenkins menu to create a new job:
+
+![](docs/images/new-item.png)
 
-![](docs/images/worddavc3d84cbb0446ba1c12cf90cedb5bd92c.png)
+2\. On the subsequent page, specify a name for your job, select **Freestyle project**, and click **OK**:
 
-4\. Click the "Configure" link.
+![](docs/images/my-job.png)
 
-![](docs/images/worddav46aff734fdfd756c63c5785d0799f97f.png)
+3\. On the resulting page, select **Subversion** in the **Source Code Management** section and enter the
+Subversion URL where the target project resides:
 
-5\. Choose "Subversion" option in the "Source Code Management" section.
+![](docs/images/source-code-management.png)
 
-![](docs/images/worddav98a0233b9c432ca9339e07ac02eceebb.png)
+  **Note**: You may need to specify user credentials to access the SVN server. To do that, click **Add** next to the **Credentials** field. 
 
-6\. Enter the subversion URL where the target project resides and press
-Tab.
+4\. In the **Build** section, click **Add build step** and select **Invoke top-level Maven targets** from the menu to add a build step:
 
-![](docs/images/worddav43269e625399dfe9a1299d654bdc03e0.png)
-Note: User credentials needs to be setup to access SVN server. Click on
-"enter credential" link to set it up if you see below error.
-![](docs/images/worddavda06736083fe517c7fbe7b3bc5fdcab1.png)
+![](docs/images/build.png)
 
-7\. To add a build step, locate the "Build' section and select "Invoke
-top-level maven targets" from the "Add build step" dropdown.
+5\. In the **Goals** field, specify *clean package*:
 
-![](docs/images/worddav63ae9fa47858a2c51daa0bf9165a371e.png)
+![](docs/images/maven.png)
 
-8\. Select "clean package" as your Goals.
+6\. Specify a post-build action. You can command Jenkins to publish test results to
+   Zephyr after the build steps are performed. To do that:  
+  * Click **Add post-build action** in the **Post-build Actions** section and select
+  **Publish test result to Zephyr Enterprise** from the menu:
 
-![](docs/images/worddav118b96747a7fbd86cc477dbb91c4944c.png)
+   ![](docs/images/post-build.png)
 
-# **Configuring Zephyr Enterprise Test Management plugin as a post-build action**
+  * Populate the following fields:
+  
+  ![](docs/images/config.png)
+  
+  * Select your Zephyr URL from the dropdown (the servers you specified on the Jenkins global configuration page are available here).
+    This automatically pulls in Zephyr projects, releases and cycles. 
+  * Select the name of your Zephyr project. This re-populates the releases. 
+  * Select a release. Selecting a release fetches all its cycles. 
+  * Select either an existing cycle or create a new one.  
+    
+    **Note**: For an existing cycle, the default cycle duration will be the value it has in Zephyr, and the cycle name prefix will be in the format *Automation_<Date><Time>*.  
```

Comment:
For existing cycle, no changes will be made to cycle i.e. no prefix will be added to cycle name. This behavior is only for creating new cycle.